apps/meteor/client/views/admin/rooms/RoomsTable.tsx

```diff
@@ -205,9 +223,21 @@ const RoomsTable = ({ reload }: { reload: MutableRefObject<() => void> }): React
 		[mediaQuery, onClick, t],
 	);
 
+	function intersect(array1: IRoom[], array2: IRoom[]) {
```

Comment:
Please, give this function a more descriptive name. Something like removeDuplicateRooms.